src/superposition_solver.py

```diff
@@ -18,7 +19,16 @@ def resolve_superposition(board, graph, cycle, quantic=True):
     # decide the coin toss
     # coin_toss_res = quantum_coin_flip()
     if quantic:
-        coin_toss_res = quantum_coin_flip()
+        list_argument = []
+        thread1 = threading.Thread(target=quantum_coin_flip, args=(list_argument,))
+        thread1.start()
+        thread1.join(2.5)
```

Comment:
Do we want to keep the timeout at 2.5 seconds, or do we want to increase it to 5, 7.5 or maybe even 10 seconds, now that it's more clear to the user why the program is unresponsive? The quantum computer might be busy with another task, and 2.5 seconds is quite short. I'd say 5 seconds might be better.

Reply:
from what ive seen, it usually takes more than 10 seconds for starmon to run the algorithms. We could change it 5 seconds and hope for the best, but waiting for more than that, but I find it annoying to wait longer than that,

Reply:
I've increased it to 5 seconds!